docs/UserGuide.md

```diff
@@ -4,339 +4,1044 @@
+
+<box type="tip" light>
+
+**Bookmark** or **create a shortcut** to this folder for easy access in the future!
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar letsgethired.jar` command to run the application. A GUI similar to the below should appear in a few seconds. Note that the app contains some sample data.<br>
+</box>
+
+__________________________________________________________________________________________________________________
+
+### Launching LetsGetHired
+
+1. Open your **Command Terminal**.
+    * On _Windows_: Press `Windows + R` keys simultaneously, type `cmd` and press `Enter`. 
+      * Alternatively, press `Windows` key, type `Terminal` and press `Enter`.
+    * On _macOS_: Press `Command + Space` to open Spotlight search, type `Terminal`, and press `Enter`.
+
+1. `cd` into your chosen home folder, and type in the following command:
+    ```
+    java -jar letsgethired.jar
+    ```
+    <box type="tip" seamless>
+    
+    If you are unfamiliar with the commands in the Command Terminal, we suggest reading [this](https://www.digitalcitizen.life/command-prompt-how-use-basic-commands/).
+    
+    </box>
+
+1. You should now see a graphical user interface (GUI) pop up on your screen (as shown in the picture below).
+
+<box type="info" light>
+
+Please note that the app comes preloaded with some **sample data**. <br>
+This is to help you get a feel for how the app works and how your entries will appear.
+
+</box>
+<br>
    
-    ![Ui](images/Ui.jpg)
+![Ui](images/Ui.jpg)
 
-1. Here's what you'll see when you start the app for the first time! We've added some intern applications already. Enter your commands in the **Command Box** on the top of the app. On the bottom left you'll see a **List View** of all your applications ordered by the date you added them. On the right, the **Select View** shows you a more in depth view of your selected application.
+| Components              | Description                                                   |
+|-------------------------|---------------------------------------------------------------|
+| Command Box             | You can type commands here and press `Enter` to execute them. |
+| Result Display Box      | Displays the result of executing a command.                   |
+| List View               | Displays a list of your internship applications.              |
+| Intern Application Card | Displays the fields related to your internship application.   |
+| Select View             | Displays more details related to your internship application. |
 
-1. Type the command in the Command Box and press `Enter` to execute it. e.g. typing **`help`** and pressing `Enter` will open the help window. Here are some example commands you can try:
+Now that you are familiar with the user interface of LetsGetHired, let's jump right into managing your 
+internship applications in this [**Tutorial**](#tutorial-managing-your-first-internship-application).
```

Comment:
Since the tutorial is right below, the link may not be necessary. Perhaps we can include a link to skip the tutorial instead? or else its fine too

Reply:
I provided the skip tutorial link at the start of the "Getting Started" section already actually, so I think this is fine!